server/superdesk/macros.py

```diff
@@ -21,6 +22,13 @@ def load_macros():
     """
     try:
         module = app.config['MACROS_MODULE']
+        already_imported = sorted(sys.modules.keys())
+        for t in already_imported:
+            if t == module:
+                m = sys.modules[t]
+                imp.reload(m)
```

Comment:
Actually should we just go and support python 3.4 and  up? Python 3.5 will be here soon so we will support the latest 2 versions.
@petrjasek Any thoughts on this?